src/modules/client/pages/ni/billing/CreditNotes.vue

```diff
@@ -603,18 +622,18 @@ export default {
         this.loading = false;
       }
     },
-    addBillingItem () {
-      this.newCreditNote.billingItemList.push({ billingItem: '', unitInclTaxes: 0, count: 1 });
+    addBillingItem (creditNote) {
```

Comment:
plutot qu de passer l'avoir en argument, tu peux utiliser `if (this.newCreditNote)` et `else if (this.editedCreditNote)` pour savoir si on utilise création ou edition
j'ai l'impression qu'on fait plus souvent comme ca, ca evite le transfert de données

Reply:
done